examples/example2.ipynb

```diff
@@ -390,15 +390,15 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Julia 1.7.3",
+   "display_name": "Julia 1.8.1",
    "language": "julia",
-   "name": "julia-1.7"
+   "name": "julia-1.8"
   },
   "language_info": {
    "file_extension": ".jl",
    "mimetype": "application/julia",
    "name": "julia",
-   "version": "1.7.3"
+   "version": "1.8.1"
```

Comment:
Why are the changes to this file part of this PR? They are unrelated, so they shouldn't come along this PR here.

Reply:
Happened automatically somehow, I guess I accidentally staged it.